app/testOnly/controllers/StartJourneyController.scala

```diff
@@ -114,14 +127,27 @@ class StartJourneyController @Inject() (
       )).map(sjResponse => Redirect(sjResponse.nextUrl.value))
     }
   }
+  /**
+   * Pretends for testing purposes that journey started from EPAYE service
+   */
+  val startJourneyEpayeEpaye: Action[AnyContent] = as.default.async { implicit request =>
+    if (hc.sessionId.isEmpty) {
+      Future.successful(Ok("Missing session id"))
+    } else {
+      journeyConnector.Epaye.startJourneyEpayeService(SjRequest.Epaye.Simple(
+        returnUrl = ReturnUrl(routes.StartJourneyController.showBtaPage.url + "?return-page"),
+        backUrl   = BackUrl(routes.StartJourneyController.showBtaPage.url + "?starting-page")
```

Comment:
shouldn't the back and return url point to the `showEpayePage` endpoint?

Reply:
updated